scripts/release_scripts/update_configs.py

```diff
@@ -167,6 +167,24 @@ def add_mailgun_api_key():
             f.write(line)
 
 
+def add_redishost():
```

Comment:
Will this be done on all other servers? Mailgun is only for prod, but this needs to be done for all deployments.

Reply:
No, it wasn't. Updated now.